node/src/components/consensus/tests/utils.rs

```diff
@@ -6,20 +6,33 @@ use once_cell::sync::Lazy;
 use casper_types::{system::auction::DelegationRate, Motes, PublicKey, SecretKey, U512};
 
 use crate::{
+    tls::{KeyFingerprint, Sha512},
     types::{
         chainspec::{AccountConfig, AccountsConfig, ValidatorConfig},
-        ActivationPoint, Chainspec, Timestamp,
+        ActivationPoint, Chainspec, NodeId, Timestamp,
     },
     utils::Loadable,
 };
 
 pub static ALICE_SECRET_KEY: Lazy<Arc<SecretKey>> =
     Lazy::new(|| Arc::new(SecretKey::ed25519_from_bytes([0; SecretKey::ED25519_LENGTH]).unwrap()));
 pub static ALICE_PUBLIC_KEY: Lazy<PublicKey> = Lazy::new(|| PublicKey::from(&**ALICE_SECRET_KEY));
+pub static ALICE_NODE_ID: Lazy<NodeId> = Lazy::new(|| {
+    NodeId::from(KeyFingerprint(Sha512::new(match *ALICE_PUBLIC_KEY {
+        PublicKey::Ed25519(pub_key) => pub_key,
+        _ => panic!("ALICE_PUBLIC_KEY is Ed25519"),
+    })))
+});
 
 pub static BOB_PRIVATE_KEY: Lazy<SecretKey> =
     Lazy::new(|| SecretKey::ed25519_from_bytes([1; SecretKey::ED25519_LENGTH]).unwrap());
 pub static BOB_PUBLIC_KEY: Lazy<PublicKey> = Lazy::new(|| PublicKey::from(&*BOB_PRIVATE_KEY));
+/* pub static BOB_NODE_ID: Lazy<NodeId> = Lazy::new(|| {
```

Comment:
If we don't need this, I think we can remove it. If we need to reimplement it and do not remember the git history, it's fairly simple to infer an implementation by looking at `ALICE_NODE_ID` :)

Reply:
If we do keep it, I believe our convention is also to use line comments (which is fairly minor).